ffi/diplomat/c/include/ICU4XUnicodeScriptMapProperty.h

```diff
@@ -0,0 +1,28 @@
+#ifndef ICU4XUnicodeScriptMapProperty_H
```

Comment:
(thought)

This file is auto-generated, right?

Would it be possible to have Diplomat add a "this file was autogenerated by ..." comment at the top of generated files, to make it easier for reviewers / readers of the code?

Reply:
Yes.  That's a good idea. https://github.com/rust-diplomat/diplomat/issues/100